engineering/solutions-engineer.md

```diff
@@ -109,40 +59,10 @@ Work you will be doing:
 - Good understanding of various applied ML concepts and problems
 - Has python programming experience on machine learning based projects
 
-### 🏗️ **Technical Stack**
-
-Skit uses a microservices architecture where we employ both HTTP and GRPC to communicate between our services. 
-
-📢 **We don't expect you to have worked on all of it. We're looking for Python programmers who are ready to pick up the rest of the tech stack on the job.**
+### Technical
 
-**Languages:**
-
-- Python
-- Golang
-
-**Frameworks:**
-
-- *Python:*
-    - Django Rest Framework
-    - Celery
-- *Golang:*
-    - Gorm
-    - GorillaMux
-    
-
-**Tools/Orchestration/VCS:**
-
-- Docker
-- Kubernetes
-- Argo Workflows
-- Gitlab
-- Github
-
-### ⏭️ Next steps
-
-Sounds interesting?
-
-If you're interested in shaking hands and joining the rocketship, please reach out to one of our team members mentioned here. If you've received an email or note from us, please respond accordingly and we will take care of it.
-
-- Abhishek (abhishek.ekka@skit.ai)
----
+- Understanding of Computer Science fundamentals, using them to effectively design and write code in python.
+- Hands-on experience developing microservices/projects in Python is preferrable.
+- Hands-on experience working with RDBMS like PostgreSQL.
+- Experience working with gRPC is a plus.
+- Experience working with container platforms like Docker, Kubernetes is a plus.
```

Comment:
I think we can add 
1. Basics of cloud computing and cloud technologies like AWS, Azure is preferable 
2. Basics of any monitoring stack like Grafana, Prometheus, OTel, etc
3. SQL and SQL dbs

Reply:
could also mention "Knowledge on Lambdas is a plus"(or we could mention that knowledge on AWS stack is preferable) .Other than that, at a general level, JD looks fine and has covered all aspects

Reply:
> could also mention "Knowledge on Lambdas is a plus"(or we could mention that knowledge on AWS stack is preferable) .Other than that, at a general level, JD looks fine and has covered all aspects

> Basics of cloud computing and cloud technologies like AWS, Azure is preferable

@runprabhsharun @adarshmoni23 
This is very specific. Considering 0-2 YOE is the range of experience we're looking for, it's a long shot to know this.
Have instead added: "_Experience with AWS or any comparable cloud technology is a plus._"

> SQL and SQL dbs

@runprabhsharun  This is mentioned under:
"_Hands-on experience working with RDBMS like PostgreSQL._"

Reply:
@misterEggroll "Experience with AWS or any comparable cloud technology is a plus." -> yeah this works.